manifests/i/Iminetsoft/DarkSwitcher/1.1.8622/Iminetsoft.DarkSwitcher.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Iminetsoft.DarkSwitcher
+PackageVersion: 1.1.8622
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /CURRENTUSER /SILENT
+Installers:
+- Architecture: x86

```

Comment:
I was incorrect about the specific dependency before. I've tested this to ensure it works. This is why it switched from `Microsoft.DotNet.Runtime.6` to `Microsoft.DotNet.DesktopRuntime.6`.

Reply:
> I was incorrect about the specific dependency before. I've tested this to ensure it works. This is why it switched from `Microsoft.DotNet.Runtime.6` to `Microsoft.DotNet.DesktopRuntime.6`.

>the installer works on x86 systems, but the application requires the x64 version of the dependency, so only works on x64 systems

That's weird, because I built the executable as hybrid (x86-x64), so I thought it could run on both architectures.
I need to figure out how to get really hybrid executable.

Reply:
@iminet,

The issue is the dependency, not the executable. I'm guessing that the executable works in 32-bit systems, and that its `x64` support requires the 64-bit dependency, which works with the `x86` application too. But the `x64` dependency won't work on those 32-bit systems. 

Were the application able to use the `x86` version of the dependency, then it would work in 32-bit systems. Maybe have it use OS architecture to require the correct version of the dependency?